x-pack/plugins/ml/server/lib/request_authorization.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaRequest } from 'kibana/server';
+
+export function authorizationHeader(request: KibanaRequest) {
```

Comment:
Nit: Maybe `getAuthorizationHeader` would make its use more clear?

Reply:
changed in https://github.com/elastic/kibana/pull/70487/commits/107e5222c3f6c2e24f835f7c192bbda68986c987